setup.py

```diff
@@ -59,11 +59,14 @@ def setup_package():
             "rich",
             "pyyaml",
             "human_dates2",
+            "click",
+            "eko",
         ],
         entry_points={
             "console_scripts": [
                 "generate_pdf=banana.data.generate_pdf:generate_pdf",
```

Comment:
If there is the new CLI the old one should be removed, not to generate confusion.